src/FSharp.Control.TaskSeq.Test/TaskSeq.StateTransitionBug.Tests.CE.fs

```diff
@@ -41,7 +41,7 @@ let ``CE  empty taskSeq, GetAsyncEnumerator multiple times`` variant = task {
 [<Theory; ClassData(typeof<TestEmptyVariants>)>]
 let ``CE  empty taskSeq, GetAsyncEnumerator multiple times and then MoveNextAsync`` variant = task {
     let tskSeq = Gen.getEmptyVariant variant
-    use enumerator = tskSeq.GetAsyncEnumerator()
+    use _enumerator = tskSeq.GetAsyncEnumerator()
```

Comment:
I think F# 8+ accepts this, which is a compiler thing, so should be good:
```suggestion
    use _ = tskSeq.GetAsyncEnumerator()
```

Reply:
yes, V6 has that too - I sometimes leave a suffix,  but this one is not exacctly helpful...